vulkaninfo/vulkaninfo.c

```diff
@@ -581,7 +581,7 @@ static const char *VkPresentModeString(VkPresentModeKHR mode) {
 
 static bool CheckExtensionEnabled(const char *extension_to_check, const char **extension_list, uint32_t extension_count) {
     for (uint32_t i = 0; i < extension_count; ++i) {
-        if (!strcmp(extension_to_check, extension_list[i])) {
+        if (!strncmp(extension_to_check, extension_list[i], VK_MAX_EXTENSION_NAME_SIZE)) {
```

Comment:
I don't see why changing `strcmp()` to `strncmp()` is required here (and below)...?  It's not a bad idea in theory, but I don't think `CheckExtensionEnabled()` is ever called without a constant extension name, and I don't think this change will make a difference...?

Is it just a hygiene thing that got left around after making the real fix?

Reply:
It's mainly a hygiene thing.  I saw it and thought I would change it.
It's probably not necessary, but it's better safe than sorry.